apps/web/src/pages/auth/components/LoginForm.tsx

```diff
@@ -79,7 +79,7 @@ export function LoginForm({ email, invitationToken }: LoginFormProps) {
       }
     }
 
-    if (organizations) {
```

Comment:
Not sure why `organizations` (all user orgs) was being checked here - `currentOrganization` is more fitting, since thats basically a subset of `organizations` so if that doesn't exist, currentOrganization also doesn't exist and vice versa